src/main/java/org/spongepowered/common/item/inventory/lens/impl/comp/GridInventoryLensImpl.java

```diff
@@ -74,6 +72,7 @@ public GridInventoryLensImpl(int base, int width, int height, int rowStride, int
     @Override
     protected void init(SlotProvider<IInventory, ItemStack> slots) {
         super.init(slots, false);
+
```

Comment:
That's fine, but you shouldn't do this here because you haven't changed anything else in this class.